### PR TITLE
Create a chainguard-keys package

### PR DESCRIPTION
**Pull request by dustinkirkland**
Let's offer a "chainguard-keys" package, which adds Chainguard's public signing keys to /etc/apk/keys.  "apk add chainguard-keys" is way better than sloppy, possibly error prone wgets of .rsa.pub files from URLs (which can be typo'd or suffer lookalike domain attacks).  It also gives us a clean way to update those keys, or ship additional keys over time (via apk update / apk upgrade).

Note that this does NOT add the repos to /etc/apk/repositories.  Just puts the keys conveniently in the keyring.

### CODE REVIEW

File: chainguard-keys.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package:
+  name: chainguard-keys
+  version: 1
+  epoch: 5
```
Comment (joshrwolf) [inline chainguard-keys.yaml:4]:
```suggestion
  epoch: 0
```

File: chainguard-keys.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    runs: |
+      mkdir -p "${{targets.destdir}}"/etc/apk/keys
+      install -m644 chainguard-extras.rsa.pub "${{targets.destdir}}"/etc/apk/keys
+      install -m644 chainguard-enterprise.rsa.pub "${{targets.destdir}}"/etc/apk/keys
+
+      for i in x86_64 aarch64; do
+        mkdir -p "${{targets.destdir}}"/usr/share/apk/keys/${i}
+        install -m644 chainguard-extras.rsa.pub "${{targets.destdir}}"/usr/share/apk/keys/${i}/chainguard-extras.rsa.pub
+        install -m644 chainguard-enterprise.rsa.pub "${{targets.destdir}}"/usr/share/apk/keys/${i}/chainguard-enterprise.rsa.pub
+      done
```
Comment (mattmoor) [inline chainguard-keys.yaml:25]: I believe apko also sets up `/etc/apk/keys`, so I would be curious which one "wins" in these cases.  cc @jonjohnsonjr

Comment (mattmoor) [reply chainguard-keys.yaml:25]: Ack this is based on `wolfi-keys`, which would already have this problem (if it's a problems).

Comment (dustinkirkland) [reply chainguard-keys.yaml:25]: Yeah, I literally cloned wolfi-keys.yaml, and assumed that it does the right thing...

File: chainguard-keys.yaml (hunk 1 above)
Comment (dustinkirkland) [reply chainguard-keys.yaml:4]: Doh.  Pushed fixes.

File: chainguard-keys.yaml (hunk 2 above)
Comment (jonjohnsonjr) [reply chainguard-keys.yaml:25]: I _think_ this should be fine but I can dig in if something goes wonky.

**Comment by tuananh** [comment]
is there a mechanism to prevent maintainer from forgetting to update this pkg when rorating chainguard keys?

**Comment by rawlingsj** [comment]
> is there a mechanism to prevent maintainer from forgetting to update this pkg when rorating chainguard keys?

There's not an automated way ATM but would be something we'd need to do.  I think we can proceed with this PR.

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**